src/main/resources/templates/laDigitalAssister/reviewContactInfo.html

```diff
@@ -32,7 +41,9 @@
               <hr class="spacing-below-35 spacing-above-35">
               <b th:text="#{review-contact-info.contact-information}"></b>
               <div class="spacing-above-15">
-                <p class="spacing-below-5" th:text="${fieldData.phoneNumber}"></p>
+                <p class="spacing-below-5" th:text="#{review-contact-info.main-phone(${fieldData.containsKey('phoneNumber') ? fieldData.phoneNumber : #messages.msg('review-contact-info.not-applicable')})}"></p>
+                <p class="spacing-below-5" th:text="#{review-contact-info.cell-phone(${fieldData.containsKey('cellPhoneNumber') ? fieldData.cellPhoneNumber : #messages.msg('review-contact-info.not-applicable')})}"></p>
```

Comment:
Despite this logic, when cell phone number is missing, I don't see "n/a" displayed:

<img width="415" alt="Screenshot 2024-01-16 at 2 09 02 PM" src="https://github.com/codeforamerica/la-doc-uploader/assets/8609011/522aa19d-cacd-4b56-be55-9a9682f0b648">

Reply:
I'm not sure how much thymeleaf can handle the embedded logic in the `#{}` wrapping.  

It might be easier if this was a variable:  ```${fieldData.containsKey('cellPhoneNumber') ? fieldData.cellPhoneNumber : #messages.msg('review-contact-info.not-applicable')}```